app/models/concerns/scripts/copy.rb

```diff
@@ -26,21 +26,43 @@ def extension
   def body inclusion = false, server = nil
     body = inclusion ? '' : "#!/usr/bin/env ruby\n\n"
 
-    unless inclusion
-      body += settings
-      body += external_gem_require if server&.local?
-      body += cores_code
+    body += lang_section :headers, server  unless inclusion
```

Comment:
¿Porqué no mejor algo así?:
```suggestion
    body += headers server unless inclusion
```
Y después
```ruby
  def headers server
    method = "#{language}_headers"

    respond_to?(method, true) ? send(method, server) : ''
  end
```

Van a ser 4 métodos en lugar de uno, pero lo dejamos menos _raro_, al menos a ojo =)

Reply:
Dale :+1:

Reply:
Hay un hermoso problema con eso... `includes` es una relación jaja.